.fixtures.yml

```diff
@@ -3,5 +3,3 @@ fixtures:
   repositories:
     simplib: https://github.com/simp/pupmod-simp-simplib
     stdlib: https://github.com/simp/puppetlabs-stdlib
-  symlinks:
```

Comment:
This needs to be re-added, making a patch

Reply:
I disagree, why did it needed to be added?

Reply:
acceptance